docs/proposals/kubelet-qos.md

```diff
@@ -0,0 +1,142 @@
+# Quality of Service in Kubernetes

```

Comment:
Resource Quality of Service.

There will be other kinds of quality of service.